src/components/CriticalActionDialog/CriticalActionDialog.tsx

```diff
@@ -13,6 +14,9 @@ import './CriticalActionDialog.scss';
 const b = cn('ydb-critical-dialog');
 
 const parseError = (error: IResponseError) => {
+    if (error.data && 'issues' in error.data && error.data.issues) {
+        return <ResultIssues data={error.data} />;
```

Comment:
Looks strange... What if we remove `ydb-critical-dialog__error-icon` in case when we show issues tree?
<img width="525" alt="Screenshot 2024-10-29 at 11 45 53" src="https://github.com/user-attachments/assets/15b50282-0dd3-4c5b-8d86-53749d868ee9">

Reply:
Is seems formatting is broken when expand issues
<img width="530" alt="Screenshot 2024-10-31 at 12 35 04" src="https://github.com/user-attachments/assets/06902c5f-74b7-4478-bf19-1f096a15da54">